### PR TITLE
PT Audit changes - get last updated test run 

### PR DESCRIPTION
**Pull request by nireeshT**
This PR has below changes 
1) Code changes to get last updated test run 
2) Unit test cases

### CODE REVIEW

File: api-audit/src/main/java/com/capitalone/dashboard/evaluator/PerformanceTestResultEvaluator.java [hunk 1]
```diff
@@ -60,13 +61,22 @@ public PerformanceTestAuditResponse evaluate(CollectorItem collectorItem, long b
...
         List<TestResult> testResults = testResultRepository.findByCollectorItemIdAndTimestampIsBetweenOrderByTimestampDesc(perfItem.getId(), beginDate-1, endDate+1);
         List<PerfTest> testlist = new ArrayList<>();
 
         for (TestResult testResult : testResults) {
             if (TestSuiteType.Performance.toString().equalsIgnoreCase(testResult.getType().name())) {
                 Collection<TestCapability> testCapabilities = testResult.getTestCapabilities();
-                for (TestCapability testCapability : testCapabilities) {
+
+                if(testCapabilities !=null){
+                    Comparator<TestCapability> testCapabilityComparator = Comparator.comparing(TestCapability::getTimestamp);
```
Comment (tabladrum) [inline api-audit/src/main/java/com/capitalone/dashboard/evaluator/PerformanceTestResultEvaluator.java]: do !CollectionUtils.isEmpty()

File: api-audit/src/main/java/com/capitalone/dashboard/evaluator/PerformanceTestResultEvaluator.java [hunk 2]
```diff
@@ -60,13 +61,22 @@ public PerformanceTestAuditResponse evaluate(CollectorItem collectorItem, long b
     private PerformanceTestAuditResponse getPerformanceTestAudit(CollectorItem perfItem, long beginDate, long endDate) {
 
         PerformanceTestAuditResponse perfReviewResponse = new PerformanceTestAuditResponse();
+        if (perfItem == null) {
```
Comment (tabladrum) [inline api-audit/src/main/java/com/capitalone/dashboard/evaluator/PerformanceTestResultEvaluator.java]: Move it before line 63

Comment (tabladrum) [reply api-audit/src/main/java/com/capitalone/dashboard/evaluator/PerformanceTestResultEvaluator.java]: NVM

File: api-audit/src/main/java/com/capitalone/dashboard/evaluator/PerformanceTestResultEvaluator.java (hunk 1 above)
Comment (nireeshT) [reply api-audit/src/main/java/com/capitalone/dashboard/evaluator/PerformanceTestResultEvaluator.java]: @tabladrum  done

**Pull request merged**